db/warehouse/migrate/20240414183410_add_enrollment_slug_to_services.rb

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class AddEnrollmentSlugToServices < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured do
+      [
+        ['Enrollment', 'slug'],
+        ['Services', 'enrollment_slug'],
+        ['CustomServices', 'enrollment_slug'],
+      ].each do |table, field|
+        execute <<~SQL
+          ALTER TABLE "#{table}" ADD COLUMN #{field} VARCHAR GENERATED ALWAYS AS (data_source_id::text || ':' || "EnrollmentID") STORED
```

Comment:
not including personal id here since enrollments may have mismatched personal ids due to merge